less/_components/quicksilver/eclipsefdn-adopters.less

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018-2020 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Christopher Guindon <chris.guindon@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+.eclipsefdn-adopters {
+    @media screen and (max-width: 767px) {
+      text-align:center;
+    }
+    ul {
+      li {
+        @media screen and (max-width: 767px) {
+          &:first-child {
+            margin-top:20px;
+          }
+        }
+        margin-bottom:20px;
+      }
+    }
+    h2 {
+      border-bottom:1px solid #999;
+      padding-bottom:10px;
+      margin-bottom:20px;
+    }
+    .adopters-img {
+      max-height: 95px;
+      padding: 10px;
+      min-width: 170px;
+      max-width: 170px;
+    }
+}
```

Comment:
For this class, since many square adopters logo for IOT is already been added spaces, this would be the best to keep the current look and feel.

I'm thinking to have a check on other working groups' adopters' logos, if other logos are fine, I will move this class into only for IOT.